caddyconfig/json5/json5.go

```diff
@@ -28,6 +29,15 @@ func init() {
 // Adapter adapts JSON5 to Caddy JSON.
 type Adapter struct{}
 
+func (_ Adapter) CaddyModule() caddy.ModuleInfo {
```

Comment:
For all these `(_ Adapter)` you can just do `(Adapter)` for a cleaner look, FYI :+1:

Reply:
And actually, instead of having each adapter declare their own ModuleInfo, you could even do this for them, since you have their name and instance value... does that make sense?

Reply:
Nope, you lost me there 😅

Reply:
Oh... maybe a dumb idea / won't work. brb